README.md

```diff
@@ -94,13 +94,16 @@ The plugin supports the following configuration parameters and include either an
 
 | Key | Description | Default |  
 |-----|-------------|---------|  
-|endpoint          |  The endpoint you send data to |  `https://log-api.newrelic.com/log/v1` |
-|apiKey            |  Your New Relic Insights Insert key | (none)   |  
-|licenseKey        |  Your New relic License key         | (none)   |
-|maxBufferSize     |  The maximum size the payloads sent in bytes  | 256000 |  
-|maxRecords        |  The maximum number of records to send at a time  | 1024 |   
-|proxy             |  Optional proxy to communicate with New Relic, overrides any environment-defined one. Must follow the format `https://user:password@hostname:port`. Can be HTTP or HTTPS. | (none) |
-|ignoreSystemProxy |  Ignore any proxy defined via the `HTTP_PROXY` or `HTTPS_PROXY` environment variables. Note that if a proxy has been defined using the `proxy` parameter, this one has no effect. | false |
+|endpoint            |  The endpoint you send data to |  `https://log-api.newrelic.com/log/v1` |
+|apiKey              |  Your New Relic Insights Insert key | (none)   |  
+|licenseKey          |  Your New relic License key         | (none)   |
+|maxBufferSize       |  The maximum size the payloads sent in bytes  | 256000 |  
+|maxRecords          |  The maximum number of records to send at a time  | 1024 |   
+|proxy               |  Optional proxy to communicate with New Relic, overrides any environment-defined one. Must follow the format `https://user:password@hostname:port`. Can be HTTP or HTTPS. | (none) |
+|ignoreSystemProxy   |  Ignore any proxy defined via the `HTTP_PROXY` or `HTTPS_PROXY` environment variables. Note that if a proxy has been defined using the `proxy` parameter, this one has no effect. | false |
+|caBundleFile        |  If your https_proxy option references to a nrclient with self-signed certificates, this option allows you specify your nrclient certificate file. | (none) |
```

Comment:
Is this wording from the infra agent? My guess is that this also allows customers to use certs that are signed, but just signed by their own CA. Also, `https_proxy` doesn't seem to be a variable here. I'm not sure customers are going to know what `nrclient` is, looks like the name of a variable in the code?

I'm not sure of the exact working, but maybe something similar to:
"Specifies the certificate authority  certificate to use for _validating_ HTTPS connections. The certificate file must be in the XYZ format (maybe PEM?). If not specified, then the operating system's CA list is used (I am totally guessing here). Only used when `validateProxyCerts`  is `true`"

I don't know, just guessing a lot :) Similar comment on `caBundleDir`

Reply:
I think you guessed very well. :-) I also like this explanation better, and is in line with what the code does. Changing it right now!

Reply:
I liked your rephrasing! I rephrased it a little bit also, let me know what you think.

Reply:
BTW, I just found out: "If your https_proxy option references to a nrclient with..." ---> seems that my wonderful IDE also replaced "proxy" by "nrclient" while I was doing some variable renaming in the code. :-)